core/orchestrator.go

```diff
@@ -141,17 +141,28 @@ func (orch *orchestrator) ProcessPayment(payment net.Payment, manifestID Manifes
 			}
 		}
 
+		// Add ticket EV to credit
+		orch.node.Accounts.Credit(manifestID, new(big.Rat).Mul(new(big.Rat).SetFrac(ticket.FaceValue, big.NewInt(1)), ticket.WinProbRat()))
+
 		if won {
 			glog.V(common.DEBUG).Infof("Received winning ticket manifestID=%v recipientRandHash=%x senderNonce=%v", manifestID, ticket.RecipientRandHash, ticket.SenderNonce)
 
 			go func(ticket *pm.Ticket, sig []byte, seed *big.Int) {
 				if err := orch.node.Recipient.RedeemWinningTicket(ticket, sig, seed); err != nil {
 					glog.Errorf("error redeeming ticket manifestID=%v recipientRandHash=%x senderNonce=%v: %v", manifestID, ticket.RecipientRandHash, ticket.SenderNonce, err)
 				}
+				// Update credit balance
```

Comment:
Don't think we need to update the credit balance here

Reply:
No indeed leftover comment from a stupid thought I had to add credit after ticket is redeemed which didnt' make sense anyway.